### DISCUSSION TITLE
Bad DNS resolutions redirect to net.domain.name

### DISCUSSION THREAD

**Discussion by antranigv** [mozilla-mobile/fenix#23149] [Q&A] [answered]
Hey all,

I am running Fennec on my Android  phone, I realized that whenever I send a request to a domain that is not available (either NXDomain or ServFail) Fennec redirects me to a `net.domain.name` website.

Things that I've tried:

- Testing the same domain with another machine in the network: does not redirect, so it's not my ISPs DNS resolvers
- Testing on the same phone with another browser: does not redirect, so it's not a LineageOS or phone network issue.

I am running the following:

95.2.0 (Build #952020)
AC: 95.0.14, 4f3b0461e7
GV: 95.0.1-20211219221914
AS: 86.0.1
Installed from F-Droid.

Here's a screenshot:

![image](https://user-images.githubusercontent.com/5600749/148925276-b4df849d-3020-4b7a-8b08-a2f08773224a.png)

The domain: https://abcde.am , which does not exist
```
$ host -t A abdce.am
Host abdce.am not found: 3(NXDOMAIN)
```

The redirection link: http://net.domain.name/?subid1=abcde.am

Thanks in advance :)

**Comment by kbrosnan** [comment, marked as answer]
This is your local network or ISP abusing NXDOMAIN not a Firefox bug. You can set your device to use a DNS that respects NXDOMAIN.

**Reply by antranigv** [reply]
Hi there! after long debugging turns out it was a mis-configuration at the ISP. Thanks for your reply, I forgot that I had this open! :-)
